README.md

```diff
@@ -294,22 +294,58 @@ To run local build of UTA you can follow these steps...
 
 1. Pick a UTA version as the base (ex. "uta_20210129b")
 
-2. Setup local database follow the steps above (`Installing with Docker`) to pull image, 
-   start and populate. __NOTE__...it does take about 5 minutes for the local database is ready.
+2. Setup local Docker container running UTA Postgresql database follow the steps above 
```

Comment:
This reads funny, maybe change follow->by following?

Reply:
Oh you are right. Ok reworded this section. Thanks @sptaylor